src/Files.App/ViewModels/UserControls/NavigationToolbarViewModel.cs

```diff
@@ -12,195 +12,270 @@
 			set
 			{
-				if (SetProperty(ref isSearchBoxVisible, value))
+				if (SetProperty(ref _IsSearchBoxVisible, value))
 					SearchButtonGlyph = value ? "\uE711" : "\uE721";
 			}
 		}
 
-		private string? pathText;
+		private string? _PathText;
 		public string? PathText
 		{
-			get => pathText;
+			get => _PathText;
 			set
 			{
-				pathText = value;
-
+				_PathText = value;
 				OnPropertyChanged(nameof(PathText));
 			}
 		}
 
-		public bool IsAppUpdated =>
-			UpdateService.IsAppUpdated;
-
-		public bool ShowHomeButton
-			=> AppearanceSettingsService.ShowHomeButton;
-
-		public bool ShowShelfPaneToggleButton
-			=> AppearanceSettingsService.ShowShelfPaneToggleButton && AppLifecycleHelper.AppEnvironment is AppEnvironment.Dev;
-
-		public ObservableCollection<NavigationBarSuggestionItem> NavigationBarSuggestions = [];
-
-		private CurrentInstanceViewModel instanceViewModel;
+		private CurrentInstanceViewModel _InstanceViewModel;
 		public CurrentInstanceViewModel InstanceViewModel
 		{
-			get => instanceViewModel;
+			get => _InstanceViewModel;
 			set
 			{
-				if (instanceViewModel?.FolderSettings is not null)
-					instanceViewModel.FolderSettings.PropertyChanged -= FolderSettings_PropertyChanged;
+				if (_InstanceViewModel?.FolderSettings is not null)
+					_InstanceViewModel.FolderSettings.PropertyChanged -= FolderSettings_PropertyChanged;
 
-				if (SetProperty(ref instanceViewModel, value) && instanceViewModel?.FolderSettings is not null)
+				if (SetProperty(ref _InstanceViewModel, value) && _InstanceViewModel?.FolderSettings is not null)
 				{
 					FolderSettings_PropertyChanged(this, new PropertyChangedEventArgs(nameof(LayoutPreferencesManager.LayoutMode)));
-					instanceViewModel.FolderSettings.PropertyChanged += FolderSettings_PropertyChanged;
+					_InstanceViewModel.FolderSettings.PropertyChanged += FolderSettings_PropertyChanged;
 				}
 			}
 		}
 
-		private Style _LayoutThemedIcon;
-		public Style LayoutThemedIcon
+		public bool IsEditModeEnabled
 		{
-			get => _LayoutThemedIcon;
+			get => ManualEntryBoxLoaded;
 			set
 			{
-				if (SetProperty(ref _LayoutThemedIcon, value))
+				if (value)
 				{
+					EditModeEnabled?.Invoke(this, EventArgs.Empty);
+
+					var visiblePath = AddressToolbar?.FindDescendant<AutoSuggestBox>(x => x.Name == "VisiblePath");
+					visiblePath?.Focus(FocusState.Programmatic);
+					visiblePath?.FindDescendant<TextBox>()?.SelectAll();
+
+					AddressBarTextEntered?.Invoke(this, new AddressBarTextEnteredEventArgs() { AddressBarTextField = visiblePath });
+				}
+				else
+				{
+					IsCommandPaletteOpen = false;
+					ManualEntryBoxLoaded = false;
+					ClickablePathLoaded = true;
 				}
 			}
 		}
 
-		private PointerRoutedEventArgs? pointerRoutedEventArgs;
+		private List<ListedItem>? _SelectedItems;
+		public List<ListedItem>? SelectedItems
+		{
+			get => _SelectedItems;
+			set
+			{
+				if (SetProperty(ref _SelectedItems, value))
+				{
+					OnPropertyChanged(nameof(CanCopy));
```

Comment:
I think we're missing CanDelete. Can you check?

Reply:
There isnt in the first place. In fact CanCopy is not used too, we can remove it safely. But I feel like I shouldnt now since this PR's scope is just reorder syntax orders based on our coding guidelines.

![image](https://github.com/user-attachments/assets/780f49c4-c0c9-42c6-bb18-6d25f2d20d42)

Reply:
It's possible this predates Actions. Feel free to remove it.

Reply:
Let me remove that next time, this diff is already cluttered enough hehe